packages/web-components/src/tooltip/tooltip.ts

```diff
@@ -0,0 +1,204 @@
+import { attr, FASTElement, nullableNumberConverter } from '@microsoft/fast-element';
+import { uniqueId } from '@microsoft/fast-web-utilities';
+
+const SUPPORTS_ANCHOR_POSITIONING = CSS.supports('anchor-name: --a');
+
+/**
+ * A Tooltip Custom HTML Element.
+ * Based on ARIA APG Tooltip Pattern {@link https://www.w3.org/WAI/ARIA/apg/patterns/tooltip/ }
+ * @public
+ */
+export class Tooltip extends FASTElement {
+  /**
+   * The attached element internals
+   */
+  public elementInternals = this.attachInternals();
+
+  /**
+   * The item ID
+   *
+   * @public
+   * @remarks
+   * HTML Attribute: id
+   */
+  @attr
+  public id: string = uniqueId('tooltip-');
+
+  /**
+   * Set the delay for the tooltip
+   */
+  @attr({ converter: nullableNumberConverter })
+  public delay?: number;
+
+  /**
+   * The default delay for the tooltip
+   * @internal
+   */
+  private defaultDelay: number = 250;
+
+  /**
+   * Set the positioning of the tooltip
+   */
+  @attr
+  public positioning?: string;
+
+  /**
+   * The id of the anchor element for the tooltip
+   */
+  @attr
+  public anchor: string = '';
```

Comment:
Wet blanket question...

Since `anchor` is in experimental web plat features in Chrome, should we guard for the instance where it's supported?

Reply:
Two paths here:

1. Disambiguate: rename the attribute to `anchors-to` and never have a collision.
2. Polyfill: detect `if (!'anchor' in HTMLElement.prototype) {}` and setup anchors otherwise do not setup anchors.

Is there a preference?

Reply:
My personal preference would be to use 2 and then guard logic that doesn't come for free with `anchor. This way  it wouldn't change the component API when `anchor` is fully supported across all browsers. Curious of thoughts from the team though.

Reply:
Added a safe guard here to only setup the anchor-name and anchor-position if `('anchor' in HTMLElement.prototype) === false`.